vscode-test-runner/runTests.js

```diff
@@ -0,0 +1,5 @@
+const vscode = require("vscode");
+
+exports.run = () => {
+  return vscode.commands.executeCommand("joyride.runCode", "(require '[integration-test.index :as it]) (it/run-all-tests)");
```

Comment:
executeCommand: does this return a promise which, when waited for, is actually the end of the command run? Or is it fire and forget?

Reply:
It returns the promise that we return from `(it/run-all-tests)`.